ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/router-pim-sparse-mode.yml

```diff
@@ -32,6 +33,7 @@ router_pim_sparse_mode:
     - name: MCAST_VRF1
       ipv4:
         bfd: true
+        make_before_break: false
```

Comment:
please add a case for make_before_break: true in next vrf to improve the coverage

Reply:
Thanks for reviewing Mahesh, I have added the test coverage for `make_before_break: true`